packages/e2e-test-utils/src/press-key-times.js

```diff
@@ -1,31 +1,11 @@
-/**
- * External dependencies
- */
-import { times } from 'lodash';
-
-/**
- * Given an array of functions, each returning a promise, performs all
- * promises in sequence (waterfall) order.
- *
- * @param {Function[]} sequence Array of promise creators.
- *
- * @return {Promise} Promise resolving once all in the sequence complete.
- */
-async function promiseSequence( sequence ) {
-	return sequence.reduce(
-		( current, next ) => current.then( next ),
-		Promise.resolve()
-	);
-}
-
 /**
  * Presses the given keyboard key a number of times in sequence.
  *
  * @param {string} key   Key to press.
  * @param {number} count Number of times to press.
- *
- * @return {Promise} Promise resolving when key presses complete.
```

Comment:
Technically it still returns a `Promise`. Unsure if ESLint would be okay with it, given the lack of an explicit return (I think it'd been an issue in the past). 

At the very least, I wonder if `docgen` could be made smart enough to always document the `Promise` return for an `async` function (cc @nosolosw), so we could avoid what otherwise appears as a documented breaking change.

Reply:
That's even better. I will fix it 👍

Reply:
Tested adding back the `return` statement and it looks like ESLint is ok (pre-commit hook didn't report any error).

Created https://github.com/WordPress/gutenberg/issues/15645 to track Andrew's proposal. In the meantime, we may want to add the `return` param back to prevent doc changes?

Reply:
Added `return` back in 644b0bc.